src/utils.rs

```diff
@@ -28,6 +28,14 @@ pub fn is_erc20transfer_event(log: &pb::eth::Log) -> bool {
     return hex::encode(&log.topics[0]) == TRANSFER_TOPIC;
 }
 
+pub fn is_mint_event(log: &pb::eth::Log) -> bool {
+    if log.topics.len() != 3 || log.data.len() != 32 {
```

Comment:
@0xbe1 It's because of those two conditions. An ERC20 `Transfer(address indexed from, address indexed from, uint256 amount)` generates 3 topics (1 for the log identifier, 2 because there is two indexed value in the event) and 32 bytes of data (because there is 1 non indexed field that goes in the `data` field of the log and will be 32 bytes long.

You Mint event has `Mint(address,uint256,uint256)` has 1 topic (only the topic identifier, because the other fields are not indexed) and will have 96 bytes of data, 32 bytes for each argument.

So you need to change to

```suggestion
    if log.topics.len() != 1 || log.data.len() != 96 {
```

The way you are going to extract the the actual arguments will be quite different also. 

We need to add ABI decoding to reduce the burden put on developers about how decode those data.